cmd/template/cluster/provider/capa.go

```diff
@@ -0,0 +1,39 @@
+package provider
+
+import (
+	"context"
+	"io"
+
+	"github.com/giantswarm/k8sclient/v5/pkg/k8sclient"
+	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/kubectl-gs/cmd/template/cluster/provider/templates/aws"
+	"github.com/giantswarm/kubectl-gs/internal/key"
+)
+
+func WriteCAPATemplate(ctx context.Context, client k8sclient.Interface, out io.Writer, config ClusterCRsConfig) error {
+	var err error
+
+	data := struct {
+		Description       string
+		KubernetesVersion string
+		Name              string
+		Namespace         string
+		Owner             string
+		ReleaseVersion    string
+	}{
+		Description:       config.Description,
+		KubernetesVersion: "v1.19.9",
```

Comment:
I guess with this approach we do not really need to set the kubernetes version here. We could let it fill in by the webhook, right?

Reply:
True - I just preserved existing functionality / settings here.